.github/workflows/build.yml

```diff
@@ -48,4 +48,12 @@ jobs:
         run: |
           make buildx.image
           make buildx.push
-
+        
+  trivy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Scan
+      uses: docker://aquasec/trivy
+      with:
+        args: --severity CRITICAL --no-progress -t ci
```

Comment:
Dont we need to mention the image that we are scanning? 

We need to scan `openebs/linux-utils` using trivy.